vcs/gitcmd/repo.go

```diff
@@ -614,6 +628,20 @@ func (r *Repository) UpdateEverything(opt vcs.RemoteOpts) error {
 		cmd.Env = []string{"GIT_SSH=" + gitSSHWrapper}
 	}
 
+	if opt.HTTPS != nil {
```

Comment:
Fair enough, although IMO makeGitPassHelper could handle the env setup, taking cmd as an arg and setting `cmd.Env = env` internally so that it is called like this:

```
gitPassHelper, env, err := makeGitPassHelper(opt.HTTPS.Pass, cmd)
if err != nil { ... }
defer os.Remove(gitPassHelper)
```

(That would call for renaming the function `makeGitPassHelper` too.)

I feel it's even more important to not duplicate sensitive logic since then there are fewer places it can be screwed up. But I think you could go either way here, so it's totally up to you.